src/test/java/com/jcabi/aspects/ImmutableTest.java

```diff
@@ -89,6 +91,35 @@ public void catchesTypesMutableByClassInheritance() {
     }
 
     /**
+     * Test Version.
+     */
+    @Test
+    public void testVersion() {
+        try {
+            new Mutable();
+        } catch (final IllegalStateException ex) {
+            MatcherAssert.assertThat(
+                ex.getMessage(),
+                Matchers.not(
+                    Matchers.containsString(
+                        "${projectVersion}"
+                    )
+                )
+            );
+            MatcherAssert.assertThat(
+                ex.getMessage(),
+                Matchers.not(
+                    Matchers.containsString(
+                        "${buildNumber}"
+                    )
+                )
+            );
+            return;
+        }
+        MatcherAssert.assertThat("should not happen.", false);
```

Comment:
@pecko It has to be done [with ExpectedException JUnit rule](https://github.com/junit-team/junit/wiki/Exception-testing). And please address all your comments by starting them with someone's username, that's our rule.